os/using-systemd-and-udev-rules.md

```diff
@@ -0,0 +1,74 @@
+# Using systemd and udev Rules 
+
+In our example we will use libvirt VM. First of all we have to create systemd unit file `/etc/systemd/system/test.service`:
```

Comment:
Can we rename this unit to something like `disk-notify` or `device-attach` or `device-notify`?